node/test/mocktests.ts

```diff
@@ -183,5 +183,56 @@ describe('Mock Tests', function () {
         
         assert(tool, "tool should not be null");
         assert(rc == 0, "rc is 0");
-    })                
+    })
+
+    it('Mock toolRunner returns success code for execTool', async () => {
+        var a: ma.TaskLibAnswers = <ma.TaskLibAnswers>{
+            "execTool": {
+                "/usr/local/bin/atool": (args: string[]) => {
+		    assert.equal(args.indexOf("--arg"), 0, "--arg should be the first element");
+		    assert.equal(args.indexOf("foo"), 1, "foo should be the second element");
```

Comment:
Your example registers a mock for task.stats, does it not? I want to register a mock for the tool being run by toolrunner. Currently you have to define the whole exec string


```
const execString = '/mocked/tools/tool' + ' tool command' +
    ` "${positionArg}"` + ` --arg1="${arg1Val}"` +
    ' --project=projectId' + ' --quiet' + ' --verbosity=info' +
    ` --version="${versionVal}"` + ' --promote' + ' --stop-previous-version';

answers.exec[execString] = {
    "code": 1,
    "stdout": '[tool output]',
    "stderr": '[tool error]'
};
```

I would like to make that easier by having an answer that only needs to know the tool path and can check that the args are correct.

```
answers.execTool['/mocked/tools/tool'] = (args: string[]) => {
    return { code: 1, stderr: '[tool error]'}
}
```